samples/house_price_estimates/README.md

```diff
@@ -0,0 +1,109 @@
+
+The read policy dictates that columns marked as having personal information should be redacted.
+
+Read Policy:
+
+```bash
+kubectl -n fybrik-system create configmap pii-read-policy --from-file=data/pii-read-policy.rego
+		
+kubectl -n fybrik-system label configmap pii-read-policy openpolicyagent.org/policy=rego
+		
+while [[ $(kubectl get cm pii-read-policy -n fybrik-system -o 'jsonpath={.metadata.annotations.openpolicyagent\.org/policy-status}') != '{"status":"ok"}' ]]; do echo "waiting for policy to be applied" && sleep 5; done
+```
+
+The write policy allows all data to be written to any location registered with Fybrik as per the [prequisits](../../get_data_endpoints/README.md).
+
+Write Policy:
+
+```bash
+kubectl -n fybrik-system create configmap allow-write-policy --from-file=data/allow-write-policy.rego
+		
+kubectl -n fybrik-system label configmap allow-write-policy openpolicyagent.org/policy=rego
+		
+while [[ $(kubectl get cm allow-write-policy -n fybrik-system -o 'jsonpath={.metadata.annotations.openpolicyagent\.org/policy-status}') != '{"status":"ok"}' ]]; do echo "waiting for policy to be applied" && sleep 5; done
+
+```
+
+### Upload Pipeline
+
+Upload either the pipeline-argo.yaml or pipeline-tekton.yaml file via the Kubeflow Pipeline GUI, depending whether you installed kubeflow pipelines with Argo (default) or with Tekton Pipelines.
+
+See slide 17 of the [demo presentation](https://drive.google.com/file/d/1xn7pGe5pEAEZxnIzDdom9r7K6s78alcP/view?usp=sharing).
+
+### Run the Pipeline
+
+Create a pipeline run via the Kubeflow Pipeline GUI, providing the following parameters:
+
+* test_dataset_id: testpii-csv
+* train_dataset_id:  trainpii-csv
+
+See slide 19 of the [demo presentation](https://drive.google.com/file/d/1xn7pGe5pEAEZxnIzDdom9r7K6s78alcP/view?usp=sharing).
+
+### View Pipeline Run Details
+
+Click on each step in the Kubeflow Pipeline GUI and view its log to see the output.  
+
+* get-data-endpoints prints and returns the virtual endpoints for the 3 dataset (test, train, results)
+* visualize-table prints a preview of the data, and you can see that the personal information has been obfuscated
+* train-model runs trains the machine learning model and then runs it on the test data, printing out a sample of the results
+* submit-result prints out the data catalog asset ID of the newly created asset containing the results
+
+See slides 21-24 of the [demo presentation](https://drive.google.com/file/d/1xn7pGe5pEAEZxnIzDdom9r7K6s78alcP/view?usp=sharing).
+
+### Making Changes to the Pipeline
+
+If you want to experiment with changing the pipeline, you will need [Kubeflow Pipelines SDK](https://www.kubeflow.org/docs/components/pipelines/sdk/install-sdk/) and its prerequisits.
+
+After making changes in either pipeline-argo.py or pipeline-tekton.py, compile the pipeline as follows:
+
+If your kubeflow pipeline deployment uses Argo Workflows (the default):
+```bash
+python3 pipeline-argo.py
```

Comment:
In order to run `pipeline-argo.py`, one needs to first `pip3 install` the `kfp` library.
I suggest adding a line explaining how to do that.
You can also consider using the `pipenv` utility, which we use in the `arrow-flight-module`. `pipenv` assumes nothing about your pre-installed python libraries, and creates a brand new python environment. The `Pipfile` file details the specific version for each package prerequisite, and (almost) guarantees uniform behavior on different machines.

Reply:
A comment for Mac M1 users, the `python3 pipeline-agro.py` (and `python3 pipeline-tekton.py`) should be run in the bash like `arch -x86_64 /bin/bash`